### PR TITLE
Content extractor ws handling

### PR DESCRIPTION
**Pull request by shabanovd**
question: should WS handling be configurable? Mean turn it on/off.

### CODE REVIEW

**Comment by ljo** [comment]
Looks good.

File: src/org/exist/util/serializer/SAXToReceiver.java [hunk 1]
```diff
@@ -34,9 +34,15 @@
 
     private Receiver receiver;
     private boolean inCDATASection = false;
+    private boolean sendWStoCharacters = false;
```
Comment (dizzzz) [inline src/org/exist/util/serializer/SAXToReceiver.java]: make consistent with "redirectIgnorableWhitespace" ? functionally it is the same...

i'd probably prefer a shorter name, avoiding "Characters". Maybe the 'invertway' notation is a bit more clear, like "ignoreWhitespace" (normally it is not)

Comment (shabanovd) [inline src/org/exist/util/serializer/SAXToReceiver.java]: @dizzzz on/off switch for ignorableWhitespace redirection to characters ... where is the problem?

**Comment by dizzzz** [comment]
I find the naming confusing/unclear ; a bit too technical

**Comment by shabanovd** [comment]
@dizzzz it have to be technical, no? My view on subject that as long as you can understand that it's ok, or we can spend few week to discuss what naming is better and find no agreement

**Comment by dizzzz** [comment]
I am  not interested into long discussions either, but I find the change not logical regardingnames; 

e.g. why "redirectIgnorableWhitespace " and "sendWStoCharacters" while they are doing exactly the same?  whilst the called method is called "characters" what the code does is actually pass whitespace data to 'something'. so IMO it is better to have the function in the variableName in stead of the name of the method that is codewise called. That is why I referred to the javadoc.

**Comment by shabanovd** [comment]
@dizzzz I'm not interesting to continue discussion. If you need different name you are welcome.

**Comment by ljo** [comment]
@dizzzz Shall I accept then?

**Comment by dizzzz** [comment]
could we at least have consisting naming then? Why is there such a negative response on my feedback?  I feel very bad about this....

**Comment by adamretter** [comment]
@dizzzz @shabanovd I think there are two separate things here and maybe you are talking at cross-purposes, I may be wrong though so please correct me:

1) The naming of the variable `sendWStoCharacters`.

2) Having two variables, which effective control the same outcome, but giving them different names i.e. `redirectIgnorableWhitespace` and `sendWStoCharacters`.

IMHO: Yes (1) could be improved from an *english* perspective, and (2) well if they are truly controlling the same thing (I am not certain), then yes it would be nice to settle on the one name.

I think @shabanovd you bowed out a little quickly, however I understand that you are busy and also agree that if @dizzzz wants to improve on this then he can of course.

@dizzzz I do think it is good to raise concerns here first as you have done. As @shabanovd has now said he is not interested in pursuing this further, it's maybe quicker for you to rebase Dmitriy's PR make changes and merge?

**Comment by dizzzz** [comment]
My proposed design is in https://github.com/eXist-db/exist/pull/981 (on purpose I did not want to put the changes directly into this PR)

**Pull request closed without merging**